src/js/background.js

```diff
@@ -459,6 +459,7 @@ Badger.prototype = {
     showCounter: true,
     showTrackingDomains: false,
     socialWidgetReplacementEnabled: true,
+    DNTEnabled: true
```

Comment:
Should use a more descriptive name like `sendDNTSignal`.

Reply:
Please insert the new setting in alphabetic order.

Reply:
Done